### PR TITLE
add cleanup crew

### PR DESCRIPTION
**Pull request by nikomatsakis**
First draft of describing the "cleanup crew" ICE-breakers team.

### CODE REVIEW

File: src/ice-breaker/cleanup-crew.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Cleanup Crew
+
+**Github Label:** [ICEBreaker-Cleanup-Crew]
+
+[ICEBreaker-LLVM]: https://github.com/rust-lang/rust/labels/ICEBreaker-Cleanup-Crew
```
Comment (lqd) [inline src/ice-breaker/cleanup-crew.md]:
```suggestion
[ICEBreaker-Cleanup-Crew]: https://github.com/rust-lang/rust/labels/ICEBreaker-Cleanup-Crew
```

Comment (mark-i-m) [reply src/ice-breaker/cleanup-crew.md]: Ping @nikomatsakis ^^^

**Comment by nikomatsakis** [comment]
OK, I pushed some basic advice about cargo-bisect-rustc. I'd like to push some advice about finding the set of PRs in a nightly, but I'm actually not sure the best way to do that.

**Comment by pnkfelix** [comment]
> OK, I pushed some basic advice about cargo-bisect-rustc. I'd like to push some advice about finding the set of PRs in a nightly, but I'm actually not sure the best way to do that.

The biggest problem here is that if you want to do something like `git log --author=bors --format=oneline COMMIT1..COMMIT2`, you need a local checkout, right? Because I would agree with that, and is a further argument for investing effort in resolving https://github.com/rust-lang/cargo-bisect-rustc/issues/40

(I'm not sure if you find the above `git log` invocation awkward; it seems pretty stream-lined to me.)

**Comment by nikomatsakis** [comment]
Yes, the biggest problem is probably needing a local checkout -- but I also think we could get more streadmlined. My ideal would be something that extracts and prints markdown with a list of PR numbers

```
* "blah blah blah", PR #123 by @so-and-so
```

File: src/ice-breaker/cleanup-crew.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+To help in figuring out the cause of a regression we have a tool
+called [cargo-bisect-rustc]. It will automatically download and test
+various builds of rustc. For recent regressions, it is even able to
+use the builds from our CI to track down the regression to a specific
+PR; for older regressions, it will simply identify a nightly.
+
+To learn to use [cargo-bisect-rustc], check out [this blog
+post][learn], which gives a quick introduction to how it works. You
+can also ask questions at the Zulip stream
+`#t-compiler/cargo-bisect-rustc`, or help in improving the tool.
```
Comment (spastorino) [inline src/ice-breaker/cleanup-crew.md]: Started reading this and I thought, I'm going to comment that we should mention the stream :smile:. This is brilliant.

File: src/ice-breaker/cleanup-crew.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+2. Execute the command `git log --author=bors --format=oneline SHA1..SHA2`
+  * This will list out all of the commits by bors, which is our merge bot
+  * Each commit corresponds to one PR, and information about the PR should be in the description
+3. Copy and paste that information into the bug report
+
+Often, just eye-balling the PR descriptions (which are included in the
+commit messages) will give you a good idea which one likely caused the
+problem. But if you're unsure feel free to just ping the compiler team
+(`@rust-lang/compiler`) or else to ping the authors of the PR
+themselves.
```
Comment (spastorino) [inline src/ice-breaker/cleanup-crew.md]: I'm not sure about this section, or at least is not clear to me why you're not relying on `cargo-bisect-rustc` for this case too. Is this for the case where the artifacts related to those PRs are not around anymore?.

**Comment by spastorino** [comment]
@nikomatsakis this is great, left a comment about the last section.

File: src/ice-breaker/cleanup-crew.md (hunk 3 above)
Comment (nikomatsakis) [reply src/ice-breaker/cleanup-crew.md]: Yes, just for that case... I'll make that clearer.

Comment (nikomatsakis) [reply src/ice-breaker/cleanup-crew.md]: done

File: src/ice-breaker/cleanup-crew.md (hunk 2 above)
Comment (lqd) [reply src/ice-breaker/cleanup-crew.md]: this here could even link to the mentioned [Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/217417-t-compiler.2Fcargo-bisect-rustc)

**Comment by spastorino** [comment]
Looks good to me now, maybe as @lqd mentioned in one case, we should link all the streams that show up there.

**Pull request merged**